### PR TITLE
Update embeds.md

### PR DESCRIPTION
**Pull request by iDrunK65**
- Change embed description limit

**Please describe the changes this PR makes and why it should be merged:**

### CODE REVIEW

File: guide/popular-topics/embeds.md [hunk 1]
```diff
@@ -402,7 +402,7 @@ There are a few limits to be aware of while planning your embeds due to the API'
 - Embed descriptions are limited to 2048 characters
 - There can be up to 25 fields
 - A field's name is limited to 256 characters and its value to 1024 characters
-- The footer text is limited to 2048 characters
+- The footer text is limited to 4096 characters
```
[the remark this answers is not in the document]
Comment (iDrunK65) [reply guide/popular-topics/embeds.md]: oops thanks I changed it

**Review by almostSouji — CHANGES_REQUESTED** [review]
> - A bot can have one embed per message
> - A webhook can have ten embeds per message

Is not true anymore. Bots can now also have up to 10 embeds per message. Could be rephrased to something like `A message can have up to 10 embeds`

**Comment by almostSouji** [comment]
can generalize the two points and replace them with one

**Comment by monbrey** [comment]
@almostSouji @iDrunK65 if this is to be a v13-based update to this page with the changes to how many embeds can be sent, then the Attaching Files section should change too

**Comment by almostSouji** [comment]
no, PR's under the v12 completion milestone are supposed to release under v12

**Review by almostSouji — APPROVED** [review]

**Pull request merged**